src/client/Project.ts

```diff
@@ -2,64 +2,80 @@ import Client, { Optional, PropertiesUpdate } from "./Client";
 import Issue, { InitialTaskProperties } from "./Task";
 import Label, { InitialLabelProperties } from "./Label";
 import Task from "./Task";
+import Status, { InitialStatusProperties } from "./Status";
 
-export interface StatusProperties {
+interface DeprecatedStatusProperties {
```

Comment:
Probably going to remove this in v2. The removal could cause problems for current users tho. Maybe create a separate app to migrate people?

OK for now